bigtable-dataflow-parent/bigtable-dataflow-seqio/README.md

```diff
@@ -31,7 +31,7 @@ Create the table in your cluster.
 On the command line:
 
 ```
-java -jar bigtable-dataflow-import-0.9.5.1-shaded.jar import \
+java -jar bigtable-dataflow-import-0.9.6-shaded.jar import \
```

Comment:
Would like to see what we're doing with the instance of `bigtable-dataflow-import` here as well.